teps/0104-tasklevel-resource-requirements.md

```diff
@@ -262,9 +291,18 @@ spec:
     - name: step-1
     - name: step-2
     - name: step-3
+```
+```
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: my-task
+spec:
+  taskRef:
+    name: my-task
   computeResources:
-    limits:
-      cpu: 2
+    requests:
+      cpu: 1.5
```

Comment:
here could still be `limits`

Reply:
thanks austin, this was a merge conflict error-- fixed now